src/features/input/InputView.vue

```diff
@@ -832,6 +844,7 @@ export default {
 
       try {
         await API.createJob(data);
+        this.overlay = !this.overlay;
         // Go to the job overview
         this.$router.push("/");
```

Comment:
Verstehe ich das hier richtig, dass nach await API.createJob(data) mit this.$route.push("/") zur Jobseite weitergeleitet wird, nachdem der createJob durchgelaufen ist? Ich weiß nicht, wie ich auf den Status zugreifen kann, um schon während await läuft, den overlay einzuschalten. Theoretisch bringt der so ja nichts mehr, weil der Job dann schon durchgelaufen ist

Reply:
Jo, mit `this.$route.push("/")` wird auf die Jobseite weitergeleitet.
Einfach vor dem try-Block.

Reply:
Wenn das vor dem try-Block steht, geht die Seite auch in den Overlay Zustand, wenn das Pushen nicht möglich war

Reply:
die Änderung des codes wird hier für den Kommentar wohl nicht übernommen :/